web/themes/joinup/prototype/sass/layouts/sections/_section--footer.sass

```diff
@@ -5,4 +5,16 @@
 // Component
 // --------------------------------------------------
 .section--footer
-  +hidden-print
+  text-align: right
+  ul
```

Comment:
Not sure this change is related to layout and grid TBH.
Also, why remove hidden print here?
At any case, rules are so generic that you could count this a hack for a specific drupal region...